api/index.js

```
@@ -12,11 +12,58 @@
 
 const app = require('express')()
 const bodyParser = require('body-parser');
+const Lob = require('lob')(process.env.LOB_TEST);
+const stripe = require("stripe")(process.env.STRIPE_TEST);
+const cors = require('cors')
+
+
+app.use(cors()); //cors hopefully
 app.use(bodyParser.json()); // support json encoded bodies
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
+
 app.post('/', (req, res) => {
-  //var value = req.body.id;
-  //res.end(`Hello from ${value}!`);
-  //TODO: DO STUFF
+  let name = req.body.name,
+      from = req.body.from,
+      street1 = req.body.street1,
+      street2 = req.body.street2,
+      city = req.body.city,
+      state = req.body.state,
+      zip = req.body.zip,
+      meme_url = req.body.url,
+      token = req.body.token;
+
+  stripe.charges.create({
+    amount: 150,
+    currency: 'usd',
+    description: 'Postcard',
+    source: token,
+  }).then(charge => {
+    console.log(charge)
+    Lob.postcards.create({
+      description: 'Demo Postcard job',
+      to: {
+        name: name,
+        address_line1: street1,
+        address_line2: street2,
+        address_city: city,
+        address_state: state,
+        address_zip: zip
+      },
+      front: 'tmpl_3dd5a5c271e43e1',
+      back: 'tmpl_730ab28adb860e7',
+      merge_variables: {
+        name: name,
+        from: from,
+        meme_url: meme_url
+      }
+    }, (err, result) => {
```

Yeah maybe return the error, so for notification of complete payment, I can notify the user of the issue. ie Lob or Stripe?

---

Yeppp makes sense!
Can you approve this PR.
Will add in the next